sdk/resourcemanager/Azure.ResourceManager/src/PsuedoGenerated/Resources/Tenant.cs

```diff
@@ -251,5 +258,278 @@ public virtual SubscriptionContainer GetSubscriptions()
         {
             return new SubscriptionContainer(this);
         }
+
+        /// <summary>
+        /// Gets the data policy manifest container for this tenant.
+        /// </summary>
+        /// <returns> A container of the data policy manifest. </returns>
+        public virtual DataPolicyManifestContainer GetDataPolicyManifests()
+        {
+            return new DataPolicyManifestContainer(this);
+        }
```

Comment:
@Yao725 you need to delegate this method in ArmClient, like this one:
https://github.com/Azure/azure-sdk-for-net/blob/fef0b61dbce12d7372f4af1bd47ed7be5fe805dd/sdk/resourcemanager/Azure.ResourceManager/src/ArmClient.cs#L341

Reply:
done